crates/noirc_evaluator/src/ssa/node.rs

```diff
@@ -204,33 +201,66 @@ impl ObjectType {
     pub fn bits(&self) -> u32 {
         match self {
             ObjectType::Boolean => 1,
-            ObjectType::NativeField => FieldElement::max_num_bits(),
             ObjectType::NotAnObject => 0,
-            ObjectType::Signed(c) => *c,
-            ObjectType::Unsigned(c) => *c,
             ObjectType::Pointer(_) => 0,
             ObjectType::Function => 0,
+            ObjectType::Numeric(numeric_type) => match numeric_type {
+                NumericType::Signed(c) | NumericType::Unsigned(c) => *c,
+                NumericType::NativeField => FieldElement::max_num_bits(),
+            },
         }
     }
 
+    /// Returns the type that represents
+    /// a field element.
+    /// The most basic/fundamental type in the language
+    pub fn native_field() -> ObjectType {
+        ObjectType::Numeric(NumericType::NativeField)
+    }
+    /// Returns a type that represents an unsigned integer
+    pub fn unsigned_integer(bit_size: u32) -> ObjectType {
+        ObjectType::Numeric(NumericType::Unsigned(bit_size))
+    }
+    /// Returns a type that represents an signed integer
+    pub fn signed_integer(bit_size: u32) -> ObjectType {
+        ObjectType::Numeric(NumericType::Signed(bit_size))
+    }
+
+    /// Returns true, if the `ObjectType`
+    /// represents a field element
```

Comment:
These comments aren't really adding much to the code. Doc comments can be useful but if they add nothing the method name and type signature don't already tell us we shouldn't pollute our code with them.

Reply:
I find Tiny docs to be helpful, especially when reading foreign codebases. For example, this is also something that Rust does, see [here](https://doc.rust-lang.org/src/alloc/vec/mod.rs.html#2060) 

In the link, its self explanatory what `is_empty` is doing when called on a vector, but the doc comments still spell it out, they come with examples, though I think thats something we can iterate on in the future where we see that as being appropriate.